docs/public/using-storage.adoc

```diff
@@ -37,4 +37,55 @@ We can delete a storage volume using `odo storage delete`
   Deleted storage store from nodejs-project-ufyy
 
   Please use `odo push` command to delete the storage from the cluster
-----
\ No newline at end of file
+----
+
+== Adding storage to specific container
+
+If your devfile has multiple containers, you can specify to which container you want the
+storage to attach to using `--container` flag in the `odo storage create` command.
```

Comment:
I think `using the --container flag` sounds better

Reply:
Updated